core/dbt/include/global_project/macros/materializations/models/incremental/merge.sql

```diff
@@ -26,7 +26,7 @@
         {% do predicates.append('FALSE') %}
     {% endif %}
 
-    {{ sql_header if sql_header is not none }}
+    {# sql_header if sql_header is not none #}
```

Comment:
(I assume this is commented out while we're working through how to plumb args/configs through)

Reply:
Yes.